docs/changehistory/NextVersion.md

```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
     alert("Integrated graphics are in use. If a discrete GPU is available, consider switching your device or browser to use it.");
 ```
 
+### Fixed inconsistent property representation in a property grid
+
+Previously when using `nestedRelatedProperties` attribute, nested related properties were merged or not merged depending on whether at least one property of an intermediate class was selected. Now in both cases properties are not merged.
+
+```json
+{
+  "id": "example",
+  "rules": [
```

Comment:
To avoid making the example so massive, could you format the rulesets in a way to save vertical space? E.g.:
```json
"rules": [{
  "ruleType": "Content",
  "specifications": [{
    "specType": "ContentInstancesOfSpecificClasses",
    "classes": { "schemaName": "BisCore", "classNames": ["GeometricModel3d"], "arePolymorphic": true }
  }]
}, {
  ...
```